tests/flow_control.rs

```diff
@@ -69,19 +69,17 @@ fn release_capacity_sends_window_update() {
             frames::headers(1)
                 .response(200)
         )
-        .send_frame(frames::data(1, &payload[0..16_384]))
-        .send_frame(frames::data(1, &payload[16_384..16_384 * 2]))
-        .send_frame(frames::data(1, &payload[16_384 * 2..16_384 * 3]))
+        .send_frame(frames::data(1, &payload[..]))
+        .send_frame(frames::data(1, &payload[..]))
+        .send_frame(frames::data(1, &payload[..]))
+        .send_frame(frames::data(1, &payload[1..]))
         .recv_frame(
-            frames::window_update(0, 32_768)
+            frames::window_update(0, 65_535)
```

Comment:
This test was checking that window updates are sent if the difference is over the ratio. The modifications to this test mean it's no longer testing that specifically, right?

This is testing that after the full window is used, sending an update of the initial window size correctly allows more data?